app/cms/utils/fetch-network-status.tsx

```diff
@@ -0,0 +1,13 @@
+import { StatuspageApiResponse } from "~/ui/design-system/src/lib/interfaces"
+import { STATUSPAGE_API_KEY, STATUSPAGE_API_URL } from "./constants"
+
+export async function fetchNetworkStatus() {
+  const response = await fetch(STATUSPAGE_API_URL, {
+    headers: {
+      Authorization: `OAuth ${STATUSPAGE_API_KEY}`,
+    },
+  }).then((r) => r.json())
+
+  const status = await Promise.all(response)
+  return status as StatuspageApiResponse[]
```

Comment:
@codingone21 i think this feedback got lost in the other PR, but i don't think Promise.all is needed here – can more simply return `response` – anyway not a big deal